fjord/feedback/templates/feedback/base.html

```diff
@@ -2,7 +2,7 @@
 {% import 'includes/common.html' as common with context %}
 
 {% block extra_headers %}
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
```

Comment:
We only need this one. The other ones don't affect feedback forms. So we should ditch those changes from the PR.